README.md

```diff
@@ -164,7 +167,7 @@ gitHub live pages
 👤 **Oluchi Theresa Ogbonna**
 
 - GitHub: [@githubhandle](https://github.com/tessyrich)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/oluchi-theresa-55b512220)
 
```

Comment:
- [ ]  Kindly ignore the highlighted code. This comment has to do with the `Key Features` section of this README.md file. Under the Key Features section, kindly include 1-3 features that you implemented in this project. 👍